src/main/java/com/derongan/minecraft/deeperworld/DeeperWorld.java

```diff
@@ -10,22 +11,19 @@
 
 public final class DeeperWorld extends JavaPlugin {
     private WorldManagerImpl worldManager;
-    private MovementListener movementListener;
-    private SectionSyncListener sectionSyncListener;
 
     @Override
     public void onEnable() {
         createConfig();
 
+        //TODO convert PlayerManager, WorldManager, and MovementListener to objects
```

Comment:
This comment makes no sense. They are objects.

Reply:
[Kotlin objects](https://kotlinlang.org/docs/tutorials/kotlin-for-py/objects-and-companion-objects.html)* (Which I now realize isn't a proper term, it just uses the object *keyword* and they're singletons, my bad!)